src/js/components/VoterGuide/GuideList.jsx

```diff
@@ -49,18 +49,26 @@ export default class GuideList extends PureComponent {
     const voterGuideList = this.sortOrganizations(nextProps.incomingVoterGuideList, ballotItemWeVoteId);
     this.setState({
       voterGuideList,
-      ballotItemWeVoteId: nextProps.ballotItemWeVoteId,
+      ballotItemWeVoteId: this.props.ballotItemWeVoteId,
```

Comment:
Hi @jarodpeachey, I can't think of any case where you would want to reassign ballotItemWeVoteId from this.props here. I will edit this back to nextProps unless you some reason for redeclaring that I'm not thinking of.